xarray/plot/utils.py

```diff
@@ -510,7 +510,7 @@ def _ensure_plottable(*args):
     Raise exception if there is anything in args that can't be plotted on an
     axis by matplotlib.
     """
-    numpy_types = [np.floating, np.integer, np.timedelta64, np.datetime64]
+    numpy_types = [np.floating, np.integer, np.timedelta64, np.datetime64, np.bool8]
```

Comment:
`np.bool`?

Reply:
`np.bool` is the python builtin, we probably need `np.bool_` instead (`np.bool8` is an alias of `np.bool_`)